.github/workflows/nuxt.yml

```
@@ -22,7 +22,7 @@ jobs:
   
   
   deploy:
-    if: github.ref == 'refs/heads/master'
+    # if: github.ref == 'refs/heads/nuxt'
```

```suggestion
    if: github.ref == 'refs/heads/nuxt'
```

Commit this before merging this PR.

For this PR I want to publish this as Github pages, but of course, after this PR, it should be triggered only for master